container-run/pkg/containerrun/cmd.go

```diff
@@ -0,0 +1,62 @@
+package containerrun
```

Comment:
I would have expected this to live below `cmd/containerrun`. I remember the cobra docs suggested a sub folder, if you intend to have multiple binaries. That's why it's `cmd/cf-operator/main.go`.